src/test/ui/terminal-width/flag-human.rs

```diff
@@ -1,4 +1,4 @@
-// compile-flags: -Z terminal-width=20
+// compile-flags: --terminal-width=20
```

Comment:
are there equivalent tests for rustdoc? If not, could you add some in `src/test/rustdoc-ui`?

Reply:
Added!